Ft sugerencias final

Laura-Mora opened the pull request:
Holis!! Ya funcionan las sugerencias, recuerden darle update al back o va a explotar

Laura-Mora committed:
Pantallas para sugerencias

Pantallas para mostrar las sugerencias de restaurantes y grupos estudiantiles, faltan pruebas!

Laura-Mora committed:
Sugerencia Grupos a un usuario

FUNCIONA!!

Laura-Mora committed:
Sugerencias Restaurantes

FUNCIONA!!

Laura-Mora committed:
Mostar info antes de editar

Laura-Mora committed:
Actualización para editar info perso

kamilo2299 reviewed (commented):
Los restaurantes recomendados salen dos veces 
![image](https://user-images.githubusercontent.com/23422907/116146142-c5012a00-a6a3-11eb-9532-be13a325c08f.png)

y la sugerencia funciona hasta que se actualiza el navegador. Si no se hace y se intenta acceder a este servicio, vuelve a aparecer el formulario de personalización


Creo que ya no se duplican los datos

Laura-Mora committed:
mini cambios sugerencias

ya no se deben repetir

Laura-Mora committed:
Mini cambios cerrar el modal

Laura-Mora committed:
Se agrego toast

dhasane committed:
Usar el token para identificar el usuario en vez del id

Aproveché para organizar un poco el html, que no me gustan las
funciones (onClick) para guardar los datos ya que usando el
[(ngModel)] se me hace mucho mas organizado.

dhasane committed:
Mostrar las recomendaciones despues de cerrar el modal

dhasane committed:
Verificar tipos de datos y recargar suge grupos, da error

Laura-Mora committed:
Mini cambio

Ya se ven las recomendaciones de grupos estudiantiles

Laura-Mora committed:
flechita para ir atras

kamilo2299 reviewed (approved):
Sirve !!! (aunque con los admins no funciona :c)

dhasane commented:
es que los admins no tienen InformacionUsuario, por lo que con ellos no supone funcionar.
Los admins estan para el dia en el que hayan pantallas de admin.
